src/index.ts

```diff
@@ -1,4 +1,10 @@
 interface Fathom {
+  /**
+   * See https://usefathom.com/docs/features/exclude
+   */
+  blockTrackingForMe?: () => void;
+  enableTrackingForMe?: () => void;
```

Comment:
We can remove the `?` here, since these functions are guaranteed to be defined on `window.fathom` (once it's loaded).